js/apps/admin-ui/src/clients/add/LogoutPanel.tsx

```diff
@@ -78,7 +82,38 @@ export const LogoutPanel = ({
           }}
         />
       )}
-      {protocol === "openid-connect" && (
+      {protocol === "openid-connect" && frontchannelLogout && (
+        <FormGroup
+          label={t("frontchannelLogoutSessionRequired")}
+          labelIcon={
+            <HelpItem
+              helpText={t("frontchannelLogoutSessionRequiredHelp")}
+              fieldLabelId="frontchannelLogoutSessionRequired"
+            />
+          }
+          fieldId="frontchannelLogoutSessionRequired"
+          hasNoPaddingTop
+        >
+          <Controller
+            name={convertAttributeNameToForm<FormFields>(
+              "attributes.frontchannel.logout.session.required",
+            )}
+            defaultValue="true"
+            control={control}
+            render={({ field }) => (
+              <Switch
+                id="frontchannelLogoutSessionRequired"
+                label={t("on")}
+                labelOff={t("off")}
+                isChecked={field.value === "true"}
+                onChange={(_event, value) => field.onChange(value.toString())}
+                aria-label={t("frontchannelLogoutSessionRequired")}
+              />
+            )}
+          />
+        </FormGroup>
```

Comment:
We have a component that does all of this:
```suggestion
       <DefaultSwitchControl
          name={convertAttributeNameToForm<FormFields>(
            "attributes.frontchannel.logout.session.required",
          )}
          defaultValue="true"
          label={t("frontchannelLogoutSessionRequired")}
          labelIcon={t("frontchannelLogoutSessionRequiredHelp")}
          stringify
        />
```